package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "devDependencies": {
+    "@babel/core": "^7.18.9",
+    "@babel/eslint-parser": "^7.18.9",
+    "@babel/plugin-syntax-jsx": "^7.18.6",
+    "@babel/preset-env": "^7.18.9",
+    "@babel/preset-react": "^7.18.6",
+    "eslint": "^8.21.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-config-react-app": "^7.0.1",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-react": "^7.30.1",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  }
+}
```

Comment:
- [ ] Please add react dependencies you can create your react app using `npx create-react-app my-app`

- [ ] Please add scripts to be able to run the app they are actually added when creating the app
```javascript
"scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
```

Reply:
All changes have been applied :smile:
Thanks for the feedback